docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -690,3 +688,27 @@ spec:
   platforms: Windows
   tags: malware, hunting
   contributors: kswagler-rh
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Firewall enabled (macOS)
+  query: SELECT 1 FROM alf WHERE global_state >= 1;
+  description: "Checks if the firewall is enabled."
+  resolution: "In System Preferences, open Security & Privacy, navigate to the Firewall tab and click Turn On Firewall."
+  platforms: macOS
+  tags: hardening, compliance, built-in
```

Comment:
Totally off-topic. A customer was requesting tags as part of a query yaml. Is this a feature or are you just using it for yourself and it doesn't affect anything in the actual product?

Reply:
This file is for the standard query library on the website which does support tags, but yeah it's a great request for the product too and one I have as well 🤩

Reply:
This is the page with the tags support: https://fleetdm.com/queries